helix-term/src/commands/typed.rs

```diff
@@ -2460,3 +2472,58 @@ pub(super) fn command_mode(cx: &mut Context) {
     prompt.recalculate_completion(cx.editor);
     cx.push_layer(Box::new(prompt));
 }
+
+fn expand_args<'a>(editor: &mut Editor, args: &'a str) -> Cow<'a, str> {
+    let reg = Regex::new(r"%(\w+)\s*\{(.*)").unwrap();
```

Comment:
Don't we need to check for the last `}` too? I think this function should have some unit tests to make sure stuff works as expected.